contracts/impl/KODAV2Override.sol

```diff
@@ -0,0 +1,77 @@
+// SPDX-License-Identifier: MIT
+
+pragma solidity ^0.8.0;
+
+/// @author: knownorigin.io
+
+import "@openzeppelin/contracts/access/Ownable.sol";
+import "@openzeppelin/contracts/utils/introspection/ERC165.sol";
+import "../specs/IKODAV2Override.sol";
+
+/**
+ * @dev Implementation of KODA V2 override
+ * @notice KnownOrigin V2 (KODAV2) records expected commissions in simplistic single digit commission i.e. 100 = 10.0%, we dont store
+ *         a primary vs secondary expected commission amount so we need work this out proportionally
+ */
+contract KODAV2Override is IKODAV2Override, ERC165, Ownable {
+
+    function supportsInterface(bytes4 interfaceId) public view virtual override(ERC165) returns (bool) {
+        return interfaceId == type(IKODAV2Override).interfaceId || super.supportsInterface(interfaceId);
+    }
```

Comment:
might be missing something here? For example here is another override interface:
```
function supportsInterface(bytes4 interfaceId) public view virtual override(ERC165, IERC165) returns (bool) {
        return interfaceId == type(IEIP2981).interfaceId || interfaceId == type(IEIP2981RoyaltyOverride).interfaceId || super.supportsInterface(interfaceId);
    }
```

Reply:
just wondering if you need `IEIP2981RoyaltyOverride` or not

Reply:
Just had another look - not sure its needed as its a override for the contract and `IEIP2981RoyaltyOverride` feels more like a reference implementation which we dont need as the override for KO calls into KO for the appropriate fees.